static/index.html

```diff
@@ -3,6 +3,7 @@
   <title>CashAddress Conversion</title>
   <link rel="stylesheet" type="text/css" href="main.css">
   <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css">
```

Comment:
Was the problem just that I needed to include this css file? :/

Reply:
For the most part, yes.  It's hidden away in the docs, but this is needed to detect the css media queries...  The rest of the changes in this diff are tweaks to make it look nicer on various device sizes.